activity/src/test/java/com/amazonaws/stepfunctions/cloudformation/activity/CreateHandlerTest.java

```diff
@@ -34,7 +34,10 @@ public void setup() {
         request = ResourceHandlerRequest.<ResourceModel>builder()
                 .region(REGION)
                 .awsAccountId(AWS_ACCOUNT_ID)
-                .desiredResourceState(ResourceModel.builder().name(ACTIVITY_NAME).build())
+                .desiredResourceState(ResourceModel.builder()
+                        .name(ACTIVITY_NAME)
+                        .build()
+                )
```

Comment:
nit: avoid style changes to files where none of the logic is being modified.

having said that, should the tests have been added for ensuring all the not found exceptions get cast as expected now that we have 3 error codes that are now mapping to resource not found??

Reply:
The NotFound exceptions don't apply to the Create handler, as we don't do any sort of resource lookup or reference in that code.